src/playbin.js

```diff
@@ -0,0 +1,70 @@
+/* exported Playbin */
+
+const {Gdk, GLib, GObject, Gtk, Gst} = imports.gi;
+
+var Playbin = GObject.registerClass({
+    Properties: {
+        uri: GObject.ParamSpec.string('uri', 'URI', '',
+                GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT,
+                'file:///app/share/lockscreens/default/animation.webm'
+            ),
+    },
+    Signals: {
+        start: {},
+        done: {},
+    }
+}, class Playbin extends Gtk.Box {
+    _init(props) {
+        super._init(props);
+
+        Gst.init_check(null);
+
+        this._started = false;
+
+        this._playbin = Gst.parse_launch('playbin name=playbin video-sink=gtksink');
+        this._playbin.videoFilter = Gst.parse_bin_from_description('videobox name=videobox ! alpha method=green', true);
+        this._videobox = this._playbin.videoFilter.get_by_name('videobox');
+
+        /* TODO: calculate right autocrop values */
+        this._videobox.top = 38;
+        this._videobox.bottom = 38;
```

Comment:
Does this explain (?)

![screenshot from 2018-11-23 10-06-29 1](https://user-images.githubusercontent.com/481418/48945018-a3664c00-ef07-11e8-8644-aac435fb31bc.png)